src/apis/sourceimages.js

```diff
@@ -11,27 +11,62 @@ export default (state) => {
   /**
    * Get a list of source images.
    *
+   * By default, listing sourceimages sorts them by created date descending.
+   *
    * ```js
    * rokka.sourceimages.list('myorg')
    * 	 .then(function(result) {})
    * 	 .catch(function(err) {});
    * ```
    *
+   * Searching for images can be achieved using the `search` parameter.
+   * Supported are predefined fields like `height`, `name` etc. but also user metadata.
+   * If you search for user metadata, the field name has to be prefixed with `user:TYPE`.
```

Comment:
I see that this is done below, sorry for that. Maybe still add a more complex query as example?